viamonvif/service.go

```diff
@@ -377,6 +384,18 @@ func fetchImageFromRTSPURL(ctx context.Context, logger logging.Logger, rtspURL s
 	}
 }
 
+// generateUniqueName creates a unique name by adding timestamp and random bytes
+func generateUniqueName(prefix string) (string, error) {
```

Comment:
This is a nice helper. Could we write unit tests for it?

Reply:
Good call, added unit test